packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -1705,6 +1705,15 @@ class Route extends EmberObject implements IRoute {
     @public
   */
   disconnectOutlet(options: string | { outlet: string; parentView?: string }) {
+    deprecate('The usage of `disconnectOutlet` is deprecated.', false, {
+      id: 'router-render-methods.disconnect-outlet',
+      until: '4.0.0',
+      url: 'TODO',
+      for: 'ember-source',
+      since: {
+        enabled: '3.26.0',
```

Comment:
```suggestion
        enabled: '3.27.0',
```

Reply:
@rwjblue so isn't the current beta cycle 3.26 when we'd expect this to land?

Reply:
So, beta has already branched away from master. This happens 6 weeks before the beta is released. During those 6 weeks, when we land features on master, they are not by _default_ included in beta - they are targeting the release _after_ beta (e.g. `canary`). 

We can, and sometimes do, patch them into the beta branch (e.g. for bugfixes). For features and deprecations, we approach it on a case-by-case basis, but we tend to avoid it unless there is a good reason and it's not a high risk change. In this case, I think it's reasonable to not backport, so we would be targeting master, which is currently `3.27.0`.

Reply:
I've decided to break the `disconnectOutlet` deprecation out into https://github.com/emberjs/ember.js/pull/19407. The `render` and `renderTemplate` deprecations are taking longer than I expected (lots of tests to fix).